docs/design/mono/diagnostics-tracing.md

```diff
@@ -454,6 +468,33 @@ won't get any additional events, just trigger a heap dump.
 
 Combining different sessions including different GC information opens up ability to track all GC allocations during a specific time period (to reduce size of captured data), while taking heap dumps into separate sessions, make it possible to do full analysis of GC memory increase/decrease tied allocation callstacks for individual object instances.
 
+### Tracing WebAssembly apps
+
+To trace a WebAssembly app:
+
+1. Add a `diagnostic_options` section to your .csproj specifying a `server` section with a `connect_url`:
+
+   ```xml
+    <ItemGroup>
+       <WasmExtraConfig Include="diagnostic_options" Value='
+    {
+        "server": {
+            "suspend": [true|false],
+            "connect_url": "ws://127.0.0.1:[port]/diagnostics"
+        }
+    }
+    ' />
+    </ItemGroup>
+   ```
+
+   Specify `"suspend":  true` to suspend the runtime at startup (**NOTE** this will make the browser unresponsive until a diagnostic tool connects and resumes it.)  The URL has to use the `ws://` scheme (`wss://` is WIP) the host should be a localhost, the port can be arbitrary, and the path `/diagnostics` is arbtrary, too.
+
+2. Run `dotnet-dsrouter server-websocket -ws http://127.0.0.1:[port]/diagnostics -ipcs /path/to/domain_socket` the `-ws` URL should be the same as the `connect_url` in the .csproj, but with the `http://` scheme.  The `-ipcs` option is some arbitrary path.
```

Comment:
Would it make sense to actually use the same connect_url as configured as diagnostic_options? I think that would make things clearer and reduce risk of copy/past errors and forget to change ws(s) to http(s).

Reply:
This all changed too.  It's now controlled by the `DOTNET_DiagnosticPorts` environment variable